app/components/articleSearch/index.tsx

```diff
@@ -227,9 +234,7 @@ class ArticleSearch extends React.PureComponent<ArticleSearchContainerProps, Art
             currentUser={currentUserState}
             papers={searchItemsToShow}
             isLoading={isContentLoading}
-            searchQueryText={
-              articleSearchState.searchFromSuggestion ? articleSearchState.suggestionKeyword : queryParams.query
-            }
+            searchQueryText={articleSearchState.suggestionKeyword || queryParams.query}
```

Comment:
is it okay?

Reply:
i mean `articleSearchState.suggestionKeyword` always exists?

Reply:
Yes it always exists. If there is no matching value, it goes down to "".